### DISCUSSION TITLE
Welcome to ORSSerialPort Discussions!

### DISCUSSION THREAD

**Discussion by armadsen** [armadsen/ORSSerialPort#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other developers using ORSSerialPort. We hope that you:
  * Ask for help using ORSSerialPort
 * Share ideas
 * Share what you're using ORSSerialPort to build
 * Welcome others and are open-minded. Remember that this is a community we build together!

 To get started, comment below with an introduction of yourself and tell us about what you do with ORSSerialPort.

**Comment by lordAnubis** [comment]
Hi, Thank you for this project.

I am busy to write code for communicating MacOS using Swift5 with PIC's and Arduino's ( Nano Every ) and found this very warm welcome project. I hope that there are more people using this and like you said, build a community that use mac to communicate wit external HW. 

I did have a question here, but I guess it should be at Q&A, so I did move it.
